lib/tasks/update_organogram_filenames.rb

```diff
@@ -0,0 +1,55 @@
+class UpdateOrganogramFilenames
+  def initialize
+    @old_urls = []
+    @new_urls = []
+  end
+
+  def call
+    csv_file = "./lib/tasks/old_new_urls.csv"
+
+    parse_csv(csv_file)
+    msg = replace_urls
+    puts msg
+  end
+
+  def parse_csv(csv_file)
+    puts "Parsing CSV '#{csv_file}'"
+    File.open(csv_file).readlines.each do |line|
+      urls = line.split(', ')
+      urls.last.delete! "\n"
+
+      @old_urls << urls.first
+      @new_urls << urls.last
+    end
+  end
+
+  def replace_urls
+    puts "Searching for urls containing '-posts-'..."
+
+    if @old_urls.empty?
+      "No urls to process"
+    else
+      Link.all.each do |link|
```

Comment:
If you're interested, using [`Link.find_each` is slightly more efficient here](https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each), but it probably don't make a big difference here.